test/e2e/run_networking_test.go

```diff
@@ -42,38 +42,37 @@ var _ = Describe("Podman run networking", func() {
 	})
 
 	It("podman run network connection with default bridge", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", ALPINE, "wget", "www.podman.io"})
-		session.Wait(90)
+		session := podmanTest.RunContainerWithNetworkTest("")
+		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with host", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", "--network", "host", ALPINE, "wget", "www.podman.io"})
-		session.Wait(90)
+		session := podmanTest.RunContainerWithNetworkTest("host")
+		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with default", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "default", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.RunContainerWithNetworkTest("default")
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with none", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "none", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.RunContainerWithNetworkTest("none")
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(1))
-		Expect(session.ErrorToString()).To(ContainSubstring("wget: bad address 'www.podman.io'"))
+		if _, found := os.LookupEnv("https_proxy"); found {
```

Comment:
This raised huge alarm bells for me, and I've had to spend a long time reading `curl(1)` and playing with the command. I'm 99% certain that this is wrong. It works because you have other proxy envariables set, but it's misleading, and will cause someone else to waste time in the future. Basically: this should be `http_proxy`, no `s`.

(An argument could be made that we should use https/443 in `curl` instead, but that would require testing for both lower and upper-case `https_proxy` which is wasteful. Port 80 is not going away in the foreseeable future. The only thing I'd ask is that you make the `curl` command above very, very, very clear what it's doing, to avoid someone else wasting time).